constraint_utils/constraint_basics.py

```diff
@@ -227,6 +247,15 @@ def filter_to_autosomes_par(ht: Union[hl.Table, hl.MatrixTable]) -> Union[hl.Tab
 
 def annotate_with_mu(ht: hl.Table, mutation_ht: hl.Table, output_loc: str = 'mu_snp',
                      keys: Tuple[str] = ('context', 'ref', 'alt', 'methylation_level')) -> hl.Table:
+    """Annotate SNP mutation rate for the input Table.
+
+    :param ht: Input table
```

Comment:
Let's be consistent throughout the repo with period use, also table -> Table
```suggestion
    :param ht: Input Table.
```